### PR TITLE
Add eslint and prettier

### PR DESCRIPTION
**Pull request by dachrillz**
For now we only lint prettier rules. Also, prettier config was
explicitly declared in case someone brings their own prettier config

### CODE REVIEW

**Comment by dachrillz** [comment]
#15 first step in closing this isse.

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "rangeEnd": null,
+  "rangeStart": 0,
+  "requirePragma": false,
+  "semi": true,
+  "singleAttributePerLine": false,
+  "singleQuote": false,
+  "tabWidth": 2,
+  "trailingComma": "es5",
+  "useTabs": false,
+  "vueIndentScriptAndStyle": false
```
Comment (bittrance) [inline .prettierrc:27]: Less relevant?

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "bracketSameLine": false,
+  "bracketSpacing": true,
+  "cursorOffset": -1,
+  "editorconfig": false,
+  "embeddedLanguageFormatting": "auto",
+  "endOfLine": "lf",
+  "filepath": "string",
+  "htmlWhitespaceSensitivity": "css",
+  "insertPragma": false,
+  "jsxSingleQuote": false,
```
Comment (bittrance) [inline .prettierrc:12]: Less relevant for a cli?

**Review by bittrance — APPROVED** [review]

File: .prettierrc (hunk 2 above)
Comment (dachrillz) [reply .prettierrc:12]: yes, but this is the standard prettier configuration!

**Pull request merged**